### DISCUSSION TITLE
Welcome to the Unofficial Trinity Continuum System for Foundry VTT!

### DISCUSSION THREAD

**Discussion by randomcard** [randomcard/trinity#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AJtheRonin** [comment]
Hi!
First of all thanks for this System. Me and my players are very grateful this exist since we can only play online and we were looking forward to play Trinity. 

We do have a quick question. Various elements/items have the option to select Enhancement, Complication and/or Associate Roll. 

What exactly happens when you click these? What kind of value you should enter? For example we tried to create the "Deal Damage" trick and drop it on the newly created "Shoot" action but we don't see any difference from just clicking on Aim, pick and attribute and manually keep track of the tricks. 

Any help would be appreciated.

**Reply by randomcard** [reply]
I'm glad this is useful for you! Here's a breakdown of those options:

Enhancement: When this is selected, it will enable the option to select an Enhancement value, and a short description. Anything with this selected will show up in the Roll options. I.E. You could make the Edge "Danger Sense", enable Enhancement and set it to 1, with a short description of "Navigating/Plotting". Then whenever you are putting a roll together, it will show in the Enhancement options.

Complication: Any Complications are supposed to show up automatically in the roll results. That way the GM could quickly determine if they applied to the roll or not. It doesn't seem to be working as intended though. I'll take a look at that next time I'm working on this.

Actions/Stunts/Tricks: The Action item does the heavy lifting. (Ex: Shoot, Close Combat Strike, etc.) Adding Stunts to it makes it easier to keep track of what stunts you can use with that action. Tricks are their own thing. Keep in mind you can click the 'word bubble' icon to output an item to chat. (And if you really wanted to group a trick with an action, nothing stopping you - but make it as a stunt so it shows up properly.)

Associated Roll: If you select this, then the item will have a small dice icon next to it on the character sheet. This does two things:
1 - You can click on the dice icon to quickly launch the Roll window.
2 - You can set that Roll Window to pull up saved Roll settings. ("Unlock" the character sheet and you'll have the option to choose which saved roll to use.)

That last bit is important. For example: you could make an Action called "Shoot", and enable Associated Roll. Then put your Roll together: Dexterity, Aim, a Gun enhancement, maybe an Edge enhancement or two. Before you make that roll, save it. Then unlock the character sheet, set the stunt to use your saved roll. Next time you click on that dice icon, it will pull up the Roll with all of those settings, hopefully saving a lot of time.

A word of warning - when building the character, I have a bug with the character sheet lock/unlock flag getting messed up. If it causes problems, reload Foundry. It hasn't come up in normal play - it has something to do with adding new items to a character.

I haven't actually played my own system yet, so there could be more errors or gaps. If you run into them, please let me know.

I hope all that helps! Have fun!

**Reply by AJtheRonin** [reply]
Thank you! This is very useful.

**Comment by AzaryKnyaz** [comment]
Hello! Thanks for this System! We switched from fate to trinity and are playing on version 11
The tokens are broken. Otherwise, everything seems to be working.
Is there any way to try and edit the system to make it easier to live with? I can try to do it myself.
